qiskit/algorithms/eigensolvers/vqd.py

```diff
@@ -78,6 +78,7 @@ class VQD(VariationalAlgorithm, Eigensolver):
     of ``None``, then VQD will look to the ansatz for a preferred value, based on its
     given initial state. If the ansatz returns ``None``,
     then a random point will be generated within the parameter bounds set, as per above.
+    It is also possible to give a list of *initial points*, one for every kth eigenvalue.
```

Comment:
It might look a bit weird to print `initial point` italic but other arguments are not 🤔